QUARKUS-874.md

```diff
@@ -23,26 +30,24 @@ NOTE: We will prepare steps to get Windows 10 instance to be able to handle situ
    - New Jenkins Jobs
  - Combinations of Quarkus extensions Test Suite
    - In Github CI - for Pull Requests and Daily runs
- - OpenShift Test Suite
-   - New Jenkins Jobs on selected modules only (the ones using deployment strategies)
-
-### Platform testing
-JVM mode on is the focus for Windows platform. This RFE does not include native executable (Mandrel) or any extension that requires native libraries (like Kafka streams client).
+ - Quarkus QE Test Suite
+   - In Github CI - for Pull Requests and Daily runs
+   - New weekly Jenkins Job running against Quarkus main
```

Comment:
Do you have cost estimate for this run?

Reply:
This is not a public information. I can count it based on RHBQ TS run, but will need to put it somewhere else.

Reply:
@rsvoboda 
misc - 28 min.
spring - 41 min.
database - 2 hr 21 min.
monitoring - 41 min.
messaging - 53 min.
security - 58 min.
http - 1 hr 10 min.
root - 2 hr 41 min.

roughly 9.88 hour per week, please refer to our billing for the cost __per On Demand Windows m5zn.metal Instance Hour_, I think other items are not important to include into counting for their price is too low

Reply:
I'll add the time estimate to the plan

Reply:
We should add the cost estimates as well as the paper track leading up to us using AWS to infra repo.

Reply:
Created MR. Thanks.